gripper_minimal_service/scripts/gripper_minimal_client_v2.py

```diff
@@ -11,7 +11,7 @@ def __init__(self):
         super().__init__('mara_minimal_client')
 
         # Create a client for service "/hrim_actuation_gripper_000000000004/goal"
-        self.client = self.create_client(ControlFinger, "/hrim_actuation_gripper_000000000004/goal")
+        self.client = self.create_client(ControlFinger, "/hrim_actuator_gripper_000000000004/fingercontrol")
```

Comment:
According to the documentation this is wrong https://acutronicrobotics.com/docs/technology/hrim/model/models/actuator/gripper